gulpfile.js

```diff
@@ -107,8 +107,13 @@ var codegen = function(project, cb) {
         deleteFolderRecursive(sourcesToDelete);
     }
 
-    console.log('Generating "' + project + '" from spec file ' + specRoot + '/' + mappings[project].source);
-    var generator = '--fluent';
+    let readmeFile = specRoot + '/' + mappings[project].source;
```

Comment:
use path.join for multi platform

Reply:
There were a comment nearby: "path.join won't work if specRoot is a URL", so it uses '/'.

For now the dirty approach would need the spec on local filesystem to work. But still keep it this way. Some day we might no longer need this hacking and re-enable the URL.